v22.1/aws-dms.md

```diff
@@ -25,6 +25,18 @@ Ensure the following items are completed prior to starting this tutorial:
 - Configure a [replication instance](https://docs.aws.amazon.com/dms/latest/userguide/CHAP_ReplicationInstance.html) in AWS.
 - Ensure you have a secure, publicly available CockroachDB cluster running v22.1 GA or later.
 
+The following databases are supported for use as source databases in AWS DMS:
+
+- Oracle 
+- Microsoft SQL Server
+- Microsoft Azure SQL
+- MySQL-compatible databases (including MariaDB)
+- PostgreSQL-compatible databases (including Amazon Aurora)
+- SAP ASE 
+- MongoDB/Amazon DocumentDB (with MongoDB compatibility) 
```

Comment:
```suggestion
- MongoDB/Amazon DocumentDB (with MongoDB compatibility)
```

Reply:
Should MongoDB/Amazon DocumentDB (with MongoDB compatibility) be on two separate lines?